targets/PythonSdk/templates/API.py.ejs

```diff
@@ -0,0 +1,34 @@
+import json
+import PlayFabErrors
+import PlayFabHTTP
+import PlayFabSettings
+
+<%- generateApiSummary("", api, "description")%>
```

Comment:
Custom hasClientOptions functions should be at the top of the file, here (see below)